src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -437,6 +464,12 @@ def geop_tasks(geom, errback):
          nlcd_kfactor)
     ]
 
+    if geom.within(DRB):
+        definitions.append(('nlcd_streams',
+                            {'polygon': [geom.geojson],
+                             'vector': streams(geom, drb=True)},
+                            nlcd_streams_drb))
```

Comment:
If an AoI is within the DRB, it looks like the `nlcd_streams` task will be run twice; once in the beginning, and once at the end.